frontend/src/app/pages/marker-page/marker-page.component.ts

```diff
@@ -51,6 +53,7 @@ export class MarkerPageComponent implements OnInit {
                 this.lastSliceID = slice.index;
             }
             this.marker.feedData(slice);
+            this.downloadingScanInProgress = false;
```

Comment:
For me it looks like spinner will be visible only when user is waiting for first slice to dowload. What about the situation where connection is laggy in the middle of labeling process and user must wait some time for new slices? (for ex. by sliding to the top or bottom of slider).